android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -669,6 +673,55 @@ public void openURL(String url, ReadableMap options) {
         mActivity.startActivity(intent);
     }
 
+  @ReactMethod
+    public void getBranchQRCode(ReadableMap branchQRCodeSettingsMap, ReadableMap branchUniversalObjectMap, ReadableMap linkPropertiesMap, ReadableMap controlParamsMap, final Promise promise) {
+        
+        BranchUniversalObject branchUniversalObject = createBranchUniversalObject(branchUniversalObjectMap);
+        LinkProperties linkProperties = createLinkProperties(linkPropertiesMap, controlParamsMap);
+        BranchQRCode qrCode = createBranchQRCode(branchQRCodeSettingsMap);
+
+        try {
+            qrCode.getQRCodeAsData(getReactApplicationContext().getCurrentActivity(), branchUniversalObject, linkProperties, new BranchQRCode.BranchQRCodeDataHandler() {
+                @Override
+                public void onSuccess(byte[] qrCodeData) {
+                    String qrCodeString = Base64.encodeToString(qrCodeData, Base64.DEFAULT);
+                    promise.resolve(qrCodeString);
+                }
+    
+                @Override
+                public void onFailure(Exception e) {
+                    Log.d("Fail in main activity", String.valueOf(e));
+                    promise.reject("Failed to get QR Code", String.valueOf(e));
+                }    
+                });
+        } catch (IOException e) {
+            e.printStackTrace();
+            Log.d("Fail in main activity", String.valueOf(e));
+        }
+    }
+
+    public BranchQRCode createBranchQRCode(ReadableMap branchQRCodeSettingsMap) {
+        BranchQRCode branchQRCode = new BranchQRCode();
+
+        if (branchQRCodeSettingsMap.hasKey("codeColor")) branchQRCode.setCodeColor(branchQRCodeSettingsMap.getString("codeColor"));
+        if (branchQRCodeSettingsMap.hasKey("backgroundColor")) branchQRCode.setBackgroundColor(branchQRCodeSettingsMap.getString("backgroundColor"));
+        if (branchQRCodeSettingsMap.hasKey("centerLogo")) branchQRCode.setCenterLogo(branchQRCodeSettingsMap.getString("centerLogo"));
+        if (branchQRCodeSettingsMap.hasKey("width")) branchQRCode.setWidth(branchQRCodeSettingsMap.getInt("width"));
```

Comment:
Can you use the constants you defined previously here? Like `Defines.Jsonkey.Width.getKey()`